core/src/main/java/org/apache/accumulo/core/file/FileSKVIterator.java

```diff
@@ -35,5 +35,5 @@
   void closeDeepCopies() throws IOException;
```

Comment:
Does closeDeepCopies still need to throw an IOException, since close itself never does?